src/USDA.NET/USDA.NET.csproj

```diff
@@ -1,14 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>USDA.NET</PackageId>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageId>USDA.API</PackageId>
     <PackageVersion>0.0.1-beta</PackageVersion>
-    <Authors>ivanmartinvalle</Authors>
+    <Authors>ivanmartinvalle, eonasdan</Authors>
     <Description>.NET client for USDA APIs</Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>USDA</PackageTags>
+    <Version>1.7.0.0</Version>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <AssemblyVersion>1.7.0.0</AssemblyVersion>
+    <FileVersion>1.7.0.0</FileVersion>
+    <Product>USDA API .NET</Product>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.4-alpha1-170714" />
```

Comment:
Can you describe your use case for removing the reference to .NET's HttpClient extensions? Using the extensions allows one to now have to deal with the API serialization format implementation detail. The extensions themselves should already reference `Newtonsoft.Json`, so you should I thought still be able to utilize the `JsonProperty` attributes.

Reply:
I could not use your repo in a Xamarin forms project with that reference.